crates/node/src/network_server/metrics.rs

```diff
@@ -188,38 +188,43 @@ pub async fn render_metrics(State(state): State<NodeCtrlHandlerState>) -> String
     let manager = RocksDbManager::get();
     let all_dbs = manager.get_all_dbs();
 
-    let mut global_labels: Vec<String> = vec![];
-    for (k, v) in state.prometheus_handle.global_labels() {
-        global_labels.push(format!("{}=\"{}\"", k, formatting::sanitize_label_value(v)));
-    }
+    let global_labels = state.prometheus_handle.global_labels();
+    let global_labels_len = global_labels.len();
+    let mut labels = Vec::with_capacity(global_labels_len + 2);
+    labels.clone_from(global_labels);
+    // Push an empty db、cf label at the end of labels,
+    // so in the for loop below, we can modify for each db label.
+    labels.extend(vec![String::from("db=\"\""), String::from("cf=\"\"")]);
```

Comment:
Let's push empty strings. I would probably also move this piece to line 215 to have it closer to the for loop where it's needed. Then you also don't have to specify the length when passing in the `labels` to the subsequent function calls.